### ISSUE TITLE
Incorrect devices info parsing

### ISSUE THREAD

**Issue by uvlad7** [cocool97/adb_client#19]
IRust shell
```rust
In: :add adb_client
 Adding dep [-]
    Updating crates.io index
      Adding adb_client v1.0.0 to dependencies.
    Updating crates.io index


In: use std::net::Ipv4Addr;
In: use adb_client::*;
In: let mut connection = AdbTcpConnection::new(Ipv4Addr::from([127,0,0,1]), 5037).unwrap();
In: connection.devices()
Out: thread 'main' panicked at /home/vladimir/.local/share/rtx/installs/rust/1.77.1/registry/src/index.crates.io-6f17d22bba15001f/adb_client-1.0.0/src/models/device.rs:30:51:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

In: connection.devices()
Out: Ok([])
```

Works well with no devices, but crashes with emulator connected
```bash
$ adb devices
List of devices attached
emulator-5554	device

```

I suggest to replace `Regex::new("^(\\w+)\t(\\w+)\n?$")` with `Regex::new("^(\\S+)\t(\\w+)\n?$")` and don't use `unwrap()`.

P.S.
You can probably [use](https://stackoverflow.com/a/35169402/13500870) `lazy_static` to prevent regex compilation every time

**Comment by cocool97** [comment]
Hey, thanks for the issue you're right about the regex ! 
Maybe we should use a more precise regex to match emulators, including check for the `emulator-`. I'll take a look at it asap but you can also open a PR ;)

**Comment by cocool97** [comment]
Fixed in e0a7c9653c19f4343ad458e0e7460c3f2d24f184
